docs/api/target.rst

```diff
@@ -355,3 +355,62 @@ Internally created variables (NAME = "foo"):
 
 Internal variable names are prefixed with ``_`` to avoid collision with input parameters.
 
+.. _blt_import_library:
+
+blt_import_library
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_import_library( NAME                     <libname>
+                        DEPENDS_ON               [dep1 [dep2 ...]]
```

Comment:
Are we planning to add a `GLOBAL` parameter to `blt_import_library`?

Reply:
I was able to add this by adding the `IMPORTED` qualifier to the created libraries (in addition to `INTERFACE`).  In addition to allowing for `GLOBAL` visibility to be set, the created libraries are automatically wrapped into the libraries that depend on them, meaning that the end result is identical to the registered libraries approach.  This means that Axom is not required to namespace its exports as I previously thought.

Reply:
👍  👍